hw/formal/lec_sv2v.do

```diff
@@ -4,64 +4,45 @@
 
 // dofile for LEC script lec_sv2v
 
-//-----------------------------------------------------------------
+//-------------------------------------------------------------------------
 // read in golden (SystemVerilog) and revised (Verilog)
-//-----------------------------------------------------------------
+//-------------------------------------------------------------------------
+set parallel option -threads 8
 
-// black box all instantiated modules (so only top-module is used)
-set undefined cell black_box
-
-// flatten module ports for comparison
+// map all multi-dimensional ports (including structs) onto 1-dim. ports
 set naming rule -mdportflatten
 
-// Some of the prim_generic_ modules are instantiated using SystemVerilog
-// wildcard port bindings. These wildcard bindings are elaborated during
-// conversion. For the comparison to work, we need to read the signature of
-// these modules, but we still want to treat them as black boxes.
-add notranslate filepathnames $LEC_DIR/prim_*.*
-
-// ensure the module under test is not blackboxed
-add notranslate filepathnames $LEC_DIR/$LEC_TOP.*
-del notranslate filepathnames $LEC_DIR/$LEC_TOP.*
-
-// read golden design
-read design -golden -sv12 \
-  $LEC_DIR/top_pkg.sv \
-  $LEC_DIR/tlul_pkg.sv \
-  $LEC_DIR/*_pkg.sv \
-  $LEC_DIR/prim_*.sv \
-  $LEC_DIR/$LEC_TOP.sv
-
-// read revised design
-read design -revised -verilog \
-  $LEC_DIR/prim_*.v \
-  $LEC_DIR/$LEC_TOP.v
+read design -golden -sv12 -f flist_gold -rootonly -root $LEC_TOP
+read design -revised -sys -f flist_rev  -rootonly -root $LEC_TOP
+// TODO: instead of using switch -sys (for old SystemVerilog,
+// older than sv2009) we should use -ve (for Verilog). But
+// currently that doesn't work because sv2v doesn't translate
+// .* port connections. Is that an sv2v bug?
```

Comment:
In general, sv2v can only convert things where the relevant information can be resolved. For wildcard port connections, this means that the module (or module header) has to be in one of the input files. I recommend sticking with the strategy of converting all of the files all at once and then splitting the result. Performing the conversion this way is also faster, as sv2v does need to parse and convert the other input files every time.

Reply:
Awesome! I will try this and then update this PR. Thanks!

Reply:
I updated the PR, this works fine now, thanks!